recipes-app/node-red/files/node-red.service.tmpl

```diff
@@ -4,7 +4,9 @@ After=syslog.target network.target
 
 [Service]
 Type=idle
-ExecStart=/usr/bin/node-red -u %h/.node-red
+User=nodered
```

Comment:
This is too early, we need to resolve #281 first. We are on it.

Reply:
With the current pattern, we can still run the systemd service as root for now and simply change the node-red user dir to `${NODE_RED_HOME_DIR}`. Will change the commit accordingly.

Reply:
Reverted. Now we still use the root user, but make things more configurable. Also `%h/.node-red` is replaced with `-u ${NODE_RED_HOME_DIR}/.node-red` to make it independent of the users home.